File: wesafe.net/WeSafe.DAL.Entities/LicensePlateRestriction.cs
using WeSafe.Shared.Enumerations;

namespace WeSafe.DAL.Entities
{
    public class LicensePlateRestriction
    {
        public int Id { get; set; }

        public string LicensePlate { get; set; }

        public int DeviceId { get; set; }

        public virtual Device Device { get; set; }

        public LicensePlateType LicensePlateType { get; set; }
    }
}

File: wesafe.net/WeSafe.WebApi.Light/ClientApp/src/app/models/licensePlateRestriction.ts
export interface ILicensePlateRestriction {
    id: number;
    licensePlate: string;
    licensePlateType: string;
}

export class LicensePlateRestriction implements ILicensePlateRestriction {
    id: number;
    licensePlate: string;
    licensePlateType: string;
}

export interface ILicensePlateType {
    id: number;
    value: string;
}

export class LicensePlateType implements ILicensePlateType {
    id: number;
    value: string;
}
